packages/edit-site/src/components/page-pages/index.js

```diff
@@ -160,7 +160,19 @@ export default function PagePages() {
 						</a>
 					);
 				},
-				filters: [ 'enumeration' ],
+				filters: [
```

Comment:
In terms of the filter API, I've added `columnLabel` as a filter property, though I don't think we should do this.

Note the labels for the "Author" filter depending on where is rendered (top-level or column filter) and whether it is a `IN` filter or a `NOT IN` one:

| Top level | Column filter |
| --- | --- |
| <img width="121" alt="Captura de ecrã 2023-10-20, às 12 56 02" src="https://github.com/WordPress/gutenberg/assets/583546/9d6bc783-52cc-4ae8-b8e3-4f982a0e3ff0"> | <img width="438" alt="Captura de ecrã 2023-10-20, às 12 57 34" src="https://github.com/WordPress/gutenberg/assets/583546/b0635547-551d-4c36-8d6a-166311566755">|

One way forward could be to make the filter type more granular. Instead of having only `enumeration`, we'll have `enumeration_in`, and `enumeration_not_in`. Using the same top-level label (`Author`) the column filters would add `is` or `is not` depending on the filter's type.

Reply:
Implemented `enumeration_in` and `enumeration_not_in` as filter types at https://github.com/WordPress/gutenberg/pull/55508/commits/f42b632646f4a34265f504227bbd9032dc0b282b

Reply:
In https://github.com/WordPress/gutenberg/pull/55508/commits/f42b632646f4a34265f504227bbd9032dc0b282b I've used the `enumeration_in` `enumeration_not_in` granurality to explore how to differentiate `IN` and `NOT IN` filters in the top-level. It could be as simple as having different UI components:

<img width="586" alt="Captura de ecrã 2023-10-20, às 13 32 53" src="https://github.com/WordPress/gutenberg/assets/583546/5b8a5ed0-9c97-4ea8-8586-00b6bcf4be46">

Though it's not yet implemented, the user may show/hide the filters they prefer, so no all filters need to be rendered at the same time.

Additionally, I'd recommend exploring other design options that show both `IN` and `NOT IN` filters using the same component. If we used a single component, at the API level, it'd be still valuable to have the filter type granularity provided by `enumeration_in`/`enumeration_not_in`. I don't see anything that would need to change API wise.

---

For testing this, apply this patch to your environment (add `author_exclude` to the list of `visibleFilters`and make sure you have the UI for the dataview CPT enabled so you can delete any dataview you have saved previously):

```diff
diff --git a/lib/experimental/data-views.php b/lib/experimental/data-views.php
index e46fe36ba0..7e9416ff24 100644
--- a/lib/experimental/data-views.php
+++ b/lib/experimental/data-views.php
@@ -15,7 +15,7 @@ function _gutenberg_register_data_views_post_type() {
                        'label'        => _x( 'Dataviews', 'post type general name', 'gutenberg' ),
                        'description'  => __( 'Post which stores the different data views configurations', 'gutenberg' ),
                        'public'       => false,
-                       'show_ui'      => false,
+                       'show_ui'      => true,
                        'show_in_rest' => true,
                        'rewrite'      => false,
                        'capabilities' => array(
diff --git a/packages/edit-site/src/components/dataviews/provider.js b/packages/edit-site/src/components/dataviews/provider.js
index bfde67a18a..2bbd922c3d 100644
--- a/packages/edit-site/src/components/dataviews/provider.js
+++ b/packages/edit-site/src/components/dataviews/provider.js
@@ -32,7 +32,7 @@ const DEFAULT_VIEWS = {
                        field: 'date',
                        direction: 'desc',
                },
-               visibleFilters: [ 'search', 'author', 'status' ],
+               visibleFilters: [ 'search', 'author', 'author_exclude', 'status' ],
                // All fields are visible by default, so it's
                // better to keep track of the hidden ones.
                hiddenFields: [ 'date', 'featured-image' ],
```